webui/react/src/components/kit/Toast.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import { message, notification } from 'components/kit/internal/dialogApi';
+
+import Icon from './Icon';
+
+export type Severity = 'Info' | 'Confirm' | 'Warning' | 'Error';
```

Comment:
let's use this type everywhere instead of the raw string values

Reply:
Could you please explain?...

Reply:
I guess what I was thinking about was not actually this type as written. Something akin to `ErrorLevel`, like
```
const Severity = {
  Info: `Info`,
  Confirm: 'Confirm',
}
```
and then use `Severity.Info`, everywhere instead of `Info`, etc.

Reply:
I see, but don't think these are raw string values, that are mostly enums and we have similar usage for features [here](https://github.com/determined-ai/determined/blob/b5845b7c80cd07b568ff2ec17b72ff6e395a9542/webui/react/src/hooks/useFeature.ts#L10)

Reply:
ok yeah, i think i was just looking at it as something similar enough to `ErrorLevel` to use similar definition but I suppose as long as `severity` is typed, this is totally fine.